packages/flutter_app/lib/pages/home_page/home_page.dart

```diff
@@ -1,16 +1,55 @@
 import 'package:flutter/material.dart';
+import 'package:themes/themes.dart';
 
-import 'components/home_page_app_bar.dart';
-import 'components/home_page_body.dart';
+import '../../gen/strings.g.dart';
+import '../../router/router.dart';
+import '../../widgets/app_cached_network_image.dart';
+import 'components/home_list_card.dart';
 
 class HomePage extends StatelessWidget {
   const HomePage({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      appBar: HomePageAppBar(),
-      body: HomePageBody(),
+    final t = Translations.of(context);
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: [
+          SliverAppBar.large(
+            flexibleSpace: FlexibleSpaceBar(
+              titlePadding: const EdgeInsets.all(16),
+              title: Text(
+                t.homePage.appBar.title,
+                style: Theme.of(context).appBarTheme.titleTextStyle?.copyWith(
+                  color: context.colorScheme.onInverseSurface,
+                  shadows: [
+                    Shadow(
+                      blurRadius: 10,
+                      color: context.colorScheme.inverseSurface,
+                    ),
+                  ],
+                ),
+              ),
```

Comment:
Since the title will be on top of the image, a shadow has been added to make the text easier to read.

Reply:
![badge](https://img.shields.io/badge/review-imo-important) 
It might be a good idea to include a comment in the code explaining the reason for this!

Reply:
Thanks! Added a comment!
[7dffc79](https://github.com/altive/flutter_app_template/pull/460/commits/7dffc79c52de227d82c519d6aecc73aa1e68e02b)